io.github.mahmoudbahaa.outlook_for_linux.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.mahmoudbahaa.outlook_for_linux
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: outlook-for-linux
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
```

Comment:
does this really support wayland? If not remove like 11, otherwise line 12 should be `fallback-x11`

(I'm pretty it's the former)

Reply:
done